### PR TITLE
Updating trademarks in mega datasheets

### PR DESCRIPTION
**Pull request by jhansson-ard**
## What This PR Changes
- Updated trademarks for mega products datasheet

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/10.mega/boards/giga-r1-wifi/datasheet/datasheet.md [hunk 1]
```diff
@@ -166,27 +166,27 @@ The STM32H7 (U1) has two digital-to-analog converters (DAC) which drive the ster
...
 
 ## Board Operation
 
 ### Getting Started - IDE
 
-If you want to program your GIGA R1 WiFi while offline you need to install the Arduino® Desktop IDE **[1]**. To connect the GIGA R1 WiFi to your computer, you will need a Type-C® USB cable, which can also provide power to the board, as indicated by the LED (DL1).
+If you want to program your GIGA R1 WiFi while offline you need to install the Arduino Desktop IDE **[1]**. To connect the GIGA R1 WiFi to your computer, you will need a Type-C® USB cable, which can also provide power to the board, as indicated by the LED (DL1).
 
-### Getting Started - Arduino Web Editor
+### Getting Started - Arduino lCloud Editor
```
Comment (jacobhylen) [inline content/hardware/10.mega/boards/giga-r1-wifi/datasheet/datasheet.md:177]:
```suggestion
### Getting Started - Arduino Cloud Editor
```

File: content/hardware/10.mega/boards/mega-2560/datasheet/datasheet.md [hunk 2]
```diff
@@ -1,356 +1,358 @@
...
+- **I/O**
+    - 54 Digital
+    - 16 Analog
+    - 15 PWM Output
+
+# Contents 
+
+## The Board
+
+Mega 2560 Rev3 is a successor board of Arduino Mega, it is dedicated to applications and projects that require large number of input output pins and the use cases which need high processing power. The Mega 2560 Rev3 comes with a much larger set of IOs when we compare it with traditional Uno board considering the form factor of both the boards.
```
Comment (jacobhylen) [inline content/hardware/10.mega/boards/mega-2560/datasheet/datasheet.md:51]:
```suggestion
Mega 2560 Rev3 is a successor board of Arduino Mega, it is dedicated to applications and projects that require large number of input output pins and the use cases which need high processing power. The Mega 2560 Rev3 comes with a much larger set of IOs when we compare it with the traditional Arduino® UNO board considering the form factor of both the boards.

```

**Review by jacobhylen — CHANGES_REQUESTED** [review]
Hej @jhansson-ard 
Left some comments, please have a look and I'll re-review ☀️

**Comment by jhansson-ard** [comment]
@jacobhylen I implemented your changes ⭐

**Review by jacobhylen — APPROVED** [review]

**Pull request merged**